.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -54,19 +51,22 @@ body:
   attributes:
     label: Steps to reproduce
     description: |
-      List of steps or sample code that reproduces the issue. Having reproducible issues is a prerequisite for contributors to be able to solve them.
-      If you include a minimal reproduction project below, you can detail how to use it here.
+      List of steps or sample code that reproduces the issue.
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Minimal reproduction project (MRP)
     description: |
-      - A small Godot project which reproduces the issue, with no unnecessary files included. Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
-      - Having an MRP is very important for contributors to be able to reproduce the bug in the same way that you are experiencing it. When testing a potential fix for the issue, contributors will use the MRP to validate that the fix is working as intended.
-      - If the reproduction steps are not project dependent (e.g. the bug is visible in a brand new project), you can write "N/A" in the field.
-      - Drag and drop a ZIP archive to upload it (max 10 MB). **Do not select another field until the project is done uploading.**
-      - **Note for C# users:** If your issue is *not* C#-specific, please upload a minimal reproduction project written in GDScript. This will make it easier for contributors to reproduce the issue locally as not everyone has a .NET setup available.
+      Contributors will use the MRP to validate that the fix is working as intended.
+      - Drag and drop a ZIP archive containing a small Godot project which reproduces the issue to upload it (max 10 MB). **Do not select another field until the project is done uploading.**
+      - Make sure that the project contains the minimal amount of assets or code.
+      - Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
+      - Detail how to use your MRP if needed.
+
+      ---
```

Comment:
I don't think we should remove the request to upload in GDScript rather than C#, the workload it adds on triagers and contributors is not insignificant IMO, though the wording might be improved

Reply:
Wording should be changed to encourage GDScript but not dismiss C# completely.